src/components/notification/notification-list.component.ts

```diff
@@ -99,8 +99,38 @@ export class NotificationListComponent implements AfterViewInit, OnDestroy {
             const element = elements[i].nativeElement;
             const notification = notifications[i];
             this._renderer.setStyle(element, 'top', `${top}px`);
-            top = top + element.scrollHeight + notification.spacing;
+            top = top + this.getNotificationHeightPixels(notification, elements[i]);
+        }
+    }
+
+    private updateListPosition(elements: ElementRef[], notifications: NotificationRef[]): void {
+        if (this.position === 'bottom-left' || this.position === 'bottom-right') {
+
+            // calculate the total height of all notifications including spacing
+            this._bottom = notifications.reduce((total, notification, index) =>
+                total + this.getNotificationHeightPixels(notification, elements[index]), 0);
+        } else {
+
+            // In a top position, bottom should be unset.
+            this._bottom = undefined;
+        }
+    }
+
+    /** Get the height of the notification, including spacing if configured. */
+    private getNotificationHeightPixels(notification: NotificationRef, elementRef: ElementRef): number {
```

Comment:
```suggestion
    private getNotificationHeightInPixels(notification: NotificationRef, elementRef: ElementRef): number {
```
I think including 'in' within the name makes this more readable. Same with `getElementOuterHeightPixels` below.